outlines/accounts.md

```diff
@@ -0,0 +1,40 @@
+1. What does Meteor do for you?
+    1. Standardized concept of userId in DDP
+    2. accounts-base package that has a standard user database, and can be plugged with different login systems
+2. The fastest way to get set up for a prototype- accounts-ui
+    1. List of easy to set up login providers, code examples
+    2. Read more about accounts user interfaces later
+3. Password login
+    1. accounts-password gives you password login with username, email, or both
+    2. How to require username, email, or both
+    3. Dealing with multiple email addresses
+    4. The case insensitivity issues - working with Meteor 1.2+ password accounts. Basically, don't access Meteor.users directly.
+    5. Email flows
+        1. Enrollment email
+        2. Reset password
+        3. Verify email
+        4. How to customize emails
```

Comment:
How to generate the HTML on the server. There are a few packages floating around that people use. I'm not sure what the best one is.